### PR TITLE
chore: update pika dependency.

### PR DESCRIPTION
**Pull request by stnley**
Stackstorm currently supports Python 3.8. The pinned pika version does not support versions python >= 3.7. The current latest version (1.3.1) is fully compatible with this pack and does not include breaking changes.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-pika>=0.11.0,<0.12.0
+pika==1.3.1
```
Comment (armab) [inline requirements.txt:1]: Let's use the https://peps.python.org/pep-0440/#version-specifiers version specifier to allow pika to upgrade to a higher patch release (if any security fix comes).
```suggestion
pika~=1.3.1
```

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Looks good overall, please also bump a minor version in the `pack.yaml` and update the Changelog so the updated pack version will be released in StackStorm Exchange.

File: requirements.txt (hunk 1 above)
Comment (stnley) [reply requirements.txt]: Thanks for the tip, I have updated the version specifier.

**Comment by stnley** [comment]
I bumped the pack version as a patch considering there are no API or code changes.

File: requirements.txt (hunk 1 above)
Comment (stnley) [reply requirements.txt]: I bumped the pack version as a patch considering there are no API or code changes.

Comment (armab) [reply requirements.txt]: Sounds good, please also update the https://github.com/StackStorm-Exchange/stackstorm-rabbitmq/blob/master/CHANGES.md

Comment (stnley) [reply requirements.txt]: Sorry, forgot about the changelog! That is now updated as well

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+# 1.1.1
+- Updated to pika 1.3.x
```
Comment (armab) [inline CHANGES.md:4]:
```suggestion
- Updated pip dependency to pika `1.3.x` to support python >= 3.7
```

**Comment by arm4b** [comment]
Merged!

**Pull request merged**